### DISCUSSION TITLE
Collaboration Design Patterns > Relations between Player Actions > Complementarity

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#113] [Framework Discussions]
# Collaboration Design Patterns > Relations between Player Actions > Complementarity

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Different ship types with different mobilities, firing ranges, armor thickness, torpedo ranges, etc, complement with other ship types' characteristics
